cs/nuget/.dir-locals.el

```diff
@@ -0,0 +1,8 @@
+;;; Directory Local Variables
+;;; For more information see (info "(emacs) Directory Variables")
+
+; In the .nuspec files we want line sorting for things like <files ...> to
+; be CASE SENSITIVE, otherwise Bond.dll and Bond.xml get sorted really far
+; away from each other.
+((nil
+  (sort-fold-case)))
```

Comment:
Yes. It automatically configures case-sensitive sorting for anyone using Emacs. I'm happy to add similar Vim magic. Just let me know what it is.